.cargo/config

```diff
@@ -1,5 +1,29 @@
 [alias]
 xtask = "run --package xtask --"
+ruma-clippy = """\
+  clippy -p ruma --all-targets --all-features --quiet -- \
```

Comment:
I would actually expect `-p ruma` to exclude xtask, though it seems like there were lints that triggered in there too? In any case, I think this can now be replaced with `--workspace` since ruma-client can now also have all features enabled at once.

```suggestion
  clippy --workspace --all-targets --all-features --quiet -- \
```

Reply:
Clippy is so weird sometimes it seems like when you fix some warnings you get others. That's what seemed to happen with xtask but they were from RA.

Ill add `--workspace` :+1: